roles/vm_create/tasks/main.yml

```diff
@@ -0,0 +1,34 @@
+---
+# Copyright (c) Stephen Bates, 2022
+#
+# Create a new VM using libvirt on the target(s). This code uses ideas
+# based on [1]. If the VM is already created we just ensure it is running.
+
+- name: Get VMs list
+  community.libvirt.virt:
+    command: list_vms
+  register: existing_vms
+  changed_when: false
+
+- name: Create VM if it does not exist
+  when: vm_create_vm_name not in existing_vms.list_vms
+  block:
+
+    - name: Use virt-install-ubuntu in qemu-minimal
+      ansible.builtin.shell:
+        chdir: ~/Projects/qemu-minimal/libvirt
```

Comment:
Maybe make this parameterized? Or does qemu-setup installs the script here?

Reply:
qemu_setup puts it here. Perhaps longer term we make the Projects folder location a parameter for all the roles that reference it but that is not part of this PR.